src/org/usfirst/frc/team2706/robot/commands/IntakeCube.java

```diff
@@ -1,19 +1,51 @@
 package org.usfirst.frc.team2706.robot.commands;
 
+import java.util.function.Supplier;
+
 import org.usfirst.frc.team2706.robot.Robot;
 import org.usfirst.frc.team2706.robot.subsystems.Intake;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Command;
 
+
 public class IntakeCube extends Command {
 
     private Intake inhale;
 
+    private final Supplier<Double> speed;
+    
+    private boolean sameRatio;
     /**
      * Allows us to use the methods in 'Intake'
+     * 
+     * @param stick The joystick to use
+     * @param axis The axis to use
      */
-    public IntakeCube() {
+    public IntakeCube(Joystick stick, int axis, boolean sameRatio) {
+        this(() -> stick.getRawAxis(axis), sameRatio);
+        
+    }
+    
+    /**
+     * Allows us to use the methods in 'Intake'
+     * 
+     * @param speed The the speed
+     */
+    public IntakeCube(double speed, boolean sameRatio) {
+        this(() -> speed, sameRatio);
+    }
+    
+    /**
+     * Allows us to use the methods in 'Intake'
+     * 
+     * @param speed The supplier for the speed
+     */
+    public IntakeCube(Supplier<Double> speed, boolean sameRatio) {
```

Comment:
Oh, I see what you are doing.   I guess using a single class to do it works fine.    You want variable and static intake speeds.   I might have coded it in a different command to differentiate, but it is fine.